charts/rasa-x/values.yaml

```diff
@@ -67,6 +67,8 @@ rasa:
   initialProbeDelay: 10
   # lockStoreDatabase is the database in redis which Rasa uses to store the conversation locks
   lockStoreDatabase: "1"
+  # extra env variables
+  extraEnvs:
```

Comment:
Is this supposed to be a string or a dict / array?

Reply:
I'd suggest to give a commented out example how this value should look like

Reply:
It is intended to be a dict and yes I agree that a commented out example would make sense here.